scraper/src/main/java/com/example/cosc345/scraper/scrapers/generic/ShopifyScraper.kt

```diff
@@ -22,7 +24,8 @@ import com.example.cosc345.shared.models.*
 abstract class ShopifyScraper(
```

Comment:
In the comment above, the line under the Process header is very long. Perhaps format that onto multiple lines.

Reply:
Android Studios reformat doesn't seem to have a problem with it, so I guess it technically complies with the quasi-code standards for this project.

Reply:
Might also mess with Markdown formatting in the compiled docs

Reply:
(Ew.)

Reply:
Ew.